src/forcegraph-kapsule.js

```diff
@@ -155,6 +155,7 @@ export default Kapsule({
     linkDirectionalParticleColor: {},
     linkDirectionalParticleResolution: { default: 4 }, // how many slice segments in the particle sphere's circumference
     forceEngine: { default: 'd3' }, // d3 or ngraph
+    d3AlphaMin: { default: Number.NEGATIVE_INFINITY, triggerUpdate: false, onChange(alphaMin, state) { state.d3ForceLayout.alphaMin(alphaMin) }},
```

Comment:
Let's set the default to `0`, since negative values of alpha don't really make sense, and `0` refers to the current backwards compatible behavior.